tests/foreman/ui/test_discoveryrule.py

```diff
@@ -94,16 +93,18 @@ def test_negative_create_discovery_rule_2(self, name):
 
         """
         with Session(self.browser) as session:
-            make_discoveryrule(session, name=name,
-                               hostgroup=self.host_group.name)
-            self.assertIsNotNone(self.discoveryrules.wait_until_element(
-                common_locators['name_haserror']
-            ))
-            self.assertIsNone(self.discoveryrules.search(name))
+            for name in "", "  ":
```

Comment:
I will combine this to the other negative test so I can use invalid_values_list() helper